pkg/cassandra/nodetool/nodetool_test.go

```diff
@@ -356,3 +358,26 @@ func TestNodeToolStatus(t *testing.T) {
 		)
 	}
 }
+
+type fakeClient struct {
+}
+
+func (c *fakeClient) StorageService() (*client.StorageService, error) {
+	ss := &client.StorageService{
+		ReleaseVersion: &version.Version{
+			semver.New("1.2.3"),
+		},
+	}
+	return ss, nil
+}
+
+var _ client.Interface = &fakeClient{}
+
+func TestNodeToolVersion(t *testing.T) {
+	nt := nodetool.New(&fakeClient{})
+	ntResponse, ntErr := nt.Version()
+	if ntErr != nil {
+		t.Error(ntErr)
+	}
+	t.Log(ntResponse)
```

Comment:
We should probably verify this version is as expected, and not just that the function doesn't error. Also should we have a test for a case where the `Version` function *does* error?